addons/sale/migrations/10.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+# © 2017 bloopark systems (<http://bloopark.de>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+# backup of separator column in table sale_layout_category because it doesn't
+# exist anymore in odoo 10
+column_renames = {
```

Comment:
If you don't have installed "sale_layout" module, columns renaming and the models renaming fails, maybe, you have to check if this module is installed to execute this two functions